### DISCUSSION TITLE
Streamlit template on windows does not work

### DISCUSSION THREAD

**Discussion by JBOE22175** [explosion/spaCy#8661] [Help: Coding & Implementations] [answered]
## How to reproduce the behaviour
```
python -m spacy project clone integrations/streamlit
python -m spacy project run install
python -m spacy project run visualize
```
gives following response. Streamlit will be opened with empty screen.
```
(spacy) C:\Users\JB\Documents\PythonScripts\spaCy\streamlit>python -m spacy project run visualize

================================= visualize =================================
Running command: streamlit run scripts/visualize.py en_core_web_sm, en_core_web_lg, en_core_web_trf '"Sundar Pichai is the CEO of Google."'

  You can now view your Streamlit app in your browser.

  Local URL: http://localhost:8501
  Network URL: http://192.168.0.7:8501

Usage: visualize.py [OPTIONS] MODELS DEFAULT_TEXT
Try 'visualize.py --help' for help.

Error: Got unexpected extra arguments (en_core_web_trf "Sundar Pichai is the CEO of Google.")
```
It works fine if I change project.yml
`models: "en_core_web_sm, en_core_web_lg, en_core_web_trf"`
to 
`models: "en_core_web_trf"`

## Your Environment
```
- **spaCy version:** 3.0.6
- **Platform:** Windows-10-10.0.19041-SP0
- **Python version:** 3.9.5
- **Pipelines:** de_core_news_md (3.0.0), de_core_news_sm (3.0.0), en_core_web_lg (3.0.0), en_core_web_md (3.0.0), en_core_web_sm (3.0.0), en_core_web_trf (3.0.0)
```

**Comment by polm** [comment, marked as answer]
Ah, apologies. I can't undo this since I've moved it to a discussion, but I didn't realize the model list was in the code as opposed to something you've supplied. So this is a bug. The source is [here](https://github.com/explosion/projects/blob/16add4e390d7ca09e7cac7b666cd4a93ec39e1ba/integrations/streamlit/project.yml#L9)

The correct format is just a comma separated list without spaces:

    en_core_web_sm,en_core_web_lg,en_core_web_trf

This isn't Windows-specific. (I'm not sure how this got through testing, maybe autoformatting put the spaces in at some point?...)

Anyway, thanks for pointing this out, we'll get it fixed!

**Reply by adrianeboyd** [reply]
(This would be a good case to test "create issue from discussion", at least?)

**Reply by polm** [reply]
Oh, I forgot they added that! Let's give it a go. (Sorry for the extra notifications)

**Reply by JBOE22175** [reply]
Great, thank you. Now, it works perfectly.
